app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::ItemsController < Api::ApiController
+  respond_to :json
+
+  before_action :authenticate
+
+  def create
+
+    @list = List.find(params[:list_id])
```

Comment:
Scope this `find` query off of `@user.lists` rather than `List`. That will prevent a logged in attacker from creating items on a victim's list.